index.html

```diff
@@ -14,7 +14,46 @@
   <header class="header-container">
     <a class="mylogo" href="#">My Logo</a>
     <nav class="menu">
-      <img src="image/menu.png" alt="Menu Icon">
+      <img src="icons/menu.png" alt="Menu Icon">
     </nav>
   </header>
-</body>
\ No newline at end of file
+  <main class="home">
+    <section class="bio">
+      <h1 class="title">I’m Jawad <br>
+        Glad to see you!</h1>
+      <p class="text">
+        I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <h3 class="connect">LET'S CONNECT</h3>
+        <ul>
+          <li><a href="https://twitter.com/Nazari_Jawad58">
```

Comment:
- [ ] Kindly 😃try to increase the margin between elements here to be like the Figma Design as you see here the elements inside each other, so it will be great if you fix that to be like the Figma Design 😊

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/220176987-9486dc88-6e4b-47cb-b0ed-f374c843fec8.png)  | ![image](https://user-images.githubusercontent.com/83360636/220176940-be3bbf07-71fb-4452-a48a-810e8a0eba29.png) |

Reply:
Hi @osamaashraf6 ,

May I ask how you checked the preview of my code, as I see there is no any such an issue in my browser that you have mentioned it here.

![image](https://user-images.githubusercontent.com/115032263/220185054-5668012f-13f8-4185-bb91-eb0d2aca30ad.png)